src/components/Button.tsx

```diff
@@ -0,0 +1,134 @@
+import * as React from 'react';
+import styled, { css } from 'styled-components';
+
+export type ButtonProps = {
+  block?: boolean;
+  children?: React.ReactNode;
+  color?: 'primary' | 'secondary' | 'tertiary' | 'danger';
+  disabled?: boolean;
+  isLoading?: boolean;
```

Comment:
We should be consistent in naming our boolean props, either prefix all of them with is... or remove it everywhere (isLoading / loading, isDisabled / disabled)

Reply:
fixed